ci/sge/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:14.04
+FROM ubuntu:14.04 as base
```

Comment:
I made this a multistage build because otherwise this will generate independent layers and the pull time increases